source/includes/_zip_codes.md

```diff
@@ -0,0 +1,55 @@
+## Zip Codes
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/zip_codes/[gmaid]?[query_params] |
+
+Use GET to retrieve zip code. Data can be returned for a GMAID for a specific date range determined by start_date and end_date. Data is an array with all zip codes for, which have at least one walk in.
```

Comment:
Let's give this more context. This is a 5-digit zipcode that can be used in a zipcode filters in the `target_geofence` and `conversion_zone_geofence` report requests.